src/content/en/2021/cms.md

```diff
@@ -49,76 +46,78 @@ There is great variability in the type and scope of the support CMSs provide for
 
 When we think about CMSs, we need to account for all the components that play a role in the viability of such a system for providing a platform for publishing content on the web. All of these components form an ecosystem surrounding the CMS platform, and they include hosting providers, extension developers, development agencies, site builders, etc. Thus, when we talk about a CMS, we usually refer to both the platform itself and its surrounding ecosystem.
 
-Our definition of a CMS in this chapter uses Wappalyzer’s [definition](https://almanac.httparchive.org/en/2021/methodology#wappalyzer) of a CMS.
+Our definition of a CMS in this chapter uses <a hreflang="en" href="https://www.wappalyzer.com/technologies/cms">Wappalyzer's definition</a> of a CMS.
 
-Shopify, Magento, Webflow, and some other platforms do not appear in this chapter’s analysis, because they are not marked as a CMS in [Wappalyzer](https://www.wappalyzer.com/technologies/cms).
+We encourage CMSs to contribute to this <a hreflang="en" href="https://github.com/AliasIO/wappalyzer">open-source project</a> to improve detection and classification in the future.
 
-We encourage CMSs to contribute to this open-source [project](https://github.com/AliasIO/wappalyzer) to improve detection and classification in the future.
+Shopify, Magento, Webflow, and some other platforms do not appear in this chapter's analysis, because they are not marked as a CMS in Wappalyzer.
 
-ECommerce platforms make a substantial part of non-CMS sites and are covered in the [ECommerce chapter](https://almanac.httparchive.org/en/2021/ecommerce). For example, Shopify grew substantially in the past year and accounted for 3.7% of websites in July according to [W3Techs](https://w3techs.com/technologies/history_overview/content_management/all/q).
+Ecommerce platforms make a substantial part of non-CMS sites and are covered in the [Ecommerce chapter](./ecommerce). For example, Shopify grew substantially in the past year and accounted for 3.7% of websites in July according to <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management/all/q">W3Techs</a>.
 
 Our research identified over 200 individual CMSs, with these ranging from a single install to millions on a single CMS.
 
-Some of them are open source (e.g. WordPress, Joomla, others) and some of them are proprietary (e.g. Wix, Squarespace, others). Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
+Some of them are open source (e.g., example, WordPress and Joomla) and some of them are proprietary (e.g., Wix and Squarespace). Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
```

Comment:
@tunetheweb `e.g., example` sounds off, was this on purpose?

Reply:
Nope! Was trying to decide between "e.g., Wordpress and Joomla" and "for example, Wordpress and Joomla" (I always think the comma after e.g. looks "off", but apparently that's the correct usage! IT doesn't bother me as much with "for example").

Anyway in my dithering I left a bit of both in! Will remove.

```suggestion
Some of them are open source (e.g., WordPress and Joomla) and some of them are proprietary (e.g., Wix and Squarespace). Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
```